python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -1547,6 +1555,9 @@ async def run_block(self):
         # Basic sets of calibrations first : biases, darks, and flats.
         # After the loop is done, do defects and PTC.
         for im_type in image_types:
+            if self.certify_calib_failed:
+                break
```

Comment:
we should keep going even if certification fails.

Reply:
This is a change from current behavior, should I create a news fragment for that?